[Docs] Add example for inversing the card icon color (#401)
mckinsey/vizro: vizro-core/docs/pages/user-guides/card-button.md

Note that in the above example the first [`Card`][vizro.models.Card] navigates to an existing [`Page`][vizro.models.Page]
in the app with `path = filters-and-parameters` and the second one to an external link.

> #### Customize an icon

Reviewer: I liked the previous header before

```suggestion
#### Add an icon responsive to theme switch
```

To add an icon that is responsive to the theme switch, you will need to override the value of the [`filter` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/filter).

> The `filter` CSS property enables the application of visual effects to elements and accepts various functions as values. In the example below, we use the one of the predefined Vizro theme CSS variables `--inverse-color` that leverages CSS `invert()` function, to invert the color of the icon on theme switch. Please note, use the said CSS variable only if your icon is white.
> If your icon color is not white, we can alter the color of the `.svg` image by adding an attribute `fill="white"` directly within the SVG code. This attribute sets the fill color of the SVG graphic to white, which makes applying visual effects to the image easier.

Reviewer: ```suggestion
The `filter` CSS property lets you add visual effects to elements using different functions. In our example, we're using the `--inverse-color` CSS variable from the Vizro theme. 

It uses the  CSS `invert()` function to fliP the color of the icon when you switch themes. Please note that this only works if your initial icon has a white fill color. If your icon isn't white, you can change its color by adding `fill="white"` directly into the SVG code.
```

> Simply assign predefined variable `--inverse-color` to `filter` property in your custom css class.

Reviewer: ```suggestion
Assign the predefined CSS variable `--inverse-color` to the `filter` property of your selected icon.
```

Note that in the above example the first [`Card`][vizro.models.Card] navigates to an existing [`Page`][vizro.models.Page]
in the app with `path = filters-and-parameters` and the second one to an external link.

> #### Icon responsive to theme switch

Reviewer 2: ```suggestion
#### Add an icon responsive to theme switch
```

> To add an icon that is responsive to the theme switch, you will need to override the value of the `filter` property.

Reviewer: ```suggestion
To add an icon that is responsive to the theme switch, you will need to override the value of the [`filter` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/filter).
```

> The `filter` property allows the application of visual effects to elements and accepts various functions as parameters. In the example below, we will use `invert()` function, to invert the color of the icon on theme switch.

Reviewer 2: ```suggestion
The `filter` property enables the application of visual effects to elements and accepts various functions as parameters. In the example below, we use the `invert()` function to invert the color of the icon on theme switch.
```

> The `filter` property allows the application of visual effects to elements and accepts various functions as parameters. In the example below, we will use `invert()` function, to invert the color of the icon on theme switch.

Reviewer: ```suggestion
The `filter` CSS property allows the application of visual effects to elements and accepts various functions as values. In the example below, we will use `invert()` function, to invert the color of the icon on theme switch.
```

1. Begin by introducing a new variable into the `vizro_dark` theme.
```css
.vizro_dark {
  --icon-color: invert(100%);

> }

Reviewer: I don't think this would actually work if they only add this change to a custom CSS file. I think they would have to copy all the other values as well. Can you double-check? And add a sentence here that they need to add the CSS to their assets folder and then link to our docs on how to add CSS into the assets folder.

I think this is actually something that users might use commonly, so I would even suggest to just add that variable to our source code. Maybe call it `--inverse-color` ? And then update the instructions because then they don't need to add any custom CSS anymore.

```

> 2. Next, utilize the previously defined variable and assign it as the value for the `filter` property in your custom CSS file:

Reviewer 2: ```suggestion
2. Next, use that variable and assign it as the value for the `filter` property in your custom CSS file:
```
